docs/docs/shapes.mdx

```diff
@@ -71,39 +157,30 @@ class CardShapeUtil extends ShapeUtil<CardShape> {
 }
 ```
 
-This is a minimal `ShapeUtil`. We've given it a static property `type` that matches the type of our shape, we've provided implementations for the abstract methods `getDefaultProps`, `getBounds`, `component`, and `indicator`.
+This is a minimal `ShapeUtil`. We've given it a static property `type` that
+matches the type of our shape, we've provided implementations for the abstract
+methods `getDefaultProps`, `getBounds`, `component`, and `indicator`.
 
-We still have work to do on the `CardShapeUtil` class, but we'll come back to it later. For now, let's put the shape onto the canvas by passing it to the `<Tldraw>` component.
+We still have work to do on the `CardShapeUtil` class, but we'll come back to it
+later. For now, let's put the shape onto the canvas by passing it to the
+`<Tldraw>` component.
 
 ### Defining the shape
 
-Before we pass the shape down, we need to package it up in a way using the `defineShape` function. We can then create an array of our defined shapes and pass them into the `<Tldraw>` component's `shapes` prop. 
+Before we pass the shape down, we need to package it up in a way using the
+`defineShape` function. We can then pass an array of our defined shapes into the
+`<Tldraw>` component's `shapes` prop. 
 
 ```tsx
 import { Tldraw } from '@tldraw/tldraw'
 import '@tldraw/tldraw/tldraw.css'
 
 const MyCardShape = defineShape('card', { util: CardShapeUtil })
-const MyCustomShapes = [MyCardShape]
-
-export default function () {
-	return (
-		<div style={{ position: 'fixed', inset: 0 }}>
-			<Tldraw shapes={MyCustomShapes}/>
-		</div>
-	)
-}
-```
 
-The `defineShape` function can also be used to include a tool that we can use to create this type of shape. For now, let's create it using the `Editor` API.
```

Comment:
Removed this to reflect incoming(?) PR https://github.com/tldraw/tldraw/pull/1691#pullrequestreview-1512597329